sdk/spring/spring-cloud-azure-supported-spring.json

```diff
@@ -11,7 +11,15 @@
     "current" : false,
     "releaseStatus" : "PRERELEASE",
     "snapshot" : false,
-    "supportStatus" : "TODO",
+    "supportStatus" : null,
```

Comment:
Does `supportStatus` should be updated manually?

Reply:
now it should be